engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host-bulk-action.factory.js

```diff
@@ -26,7 +26,8 @@ angular.module('Bastion.hosts').factory('HostBulkAction',
             releaseVersion: {method: 'PUT', params: {action: 'release_version'}},
             availableIncrementalUpdates: {method: 'POST', isArray: true, params: {action: 'available_incremental_updates'}},
             moduleStreams: {method: 'POST', params: {action: 'module_streams'}},
-            traces: {method: 'POST', params: {action: 'traces'}}
+            traces: {method: 'POST', params: {action: 'traces'}},
+            resolve: {method: 'PUT', isArray: true, params: {action: 'resolve'}}
```

Comment:
let's call this something like resolveTraces since it's a little ambiguous in this context (this factory can do all sorts of stuff)

Reply:
Updated to `resolve_traces` and the other to `fetch_traces`